shell/platform/darwin/ios/ios_context.h

```diff
@@ -90,7 +90,7 @@ class IOSContext {
   ///             bound to the current
   /// thread.
   ///
-  virtual std::unique_ptr<GLContextResult> MakeCurrent() = 0;
+  virtual std::unique_ptr<GLContextResult> MakeCurrent() const = 0;
```

Comment:
This call may be technically const because it's probably editing some global state and not modifying instance variables.  But since it has side effects I'm hesitant to call it `const`.  It is conceivable that an implementor would have to edit state to implement the behavior.

Side note, the docstring isn't very good.  The brief description describes when it is called, not what the method is actually suppose to do.  You don't have to fix that if you don't want to, it was just annoying when I did the review =)